proposals/description/fip_107.ts

```diff
@@ -0,0 +1,65 @@
+import { ProposalDescription } from '@custom-types/types';
+
+const fip_107: ProposalDescription = {
+  title: 'FIP-107: Acquire OHM by selling $10M of ETH',
+  commands: [
+    {
+      target: 'gohm',
+      values: '0',
+      method: 'transfer(address,uint256)',
+      arguments: ['{ethTogOhmLBPSwapper}', '200000000000000000000'],
```

Comment:
That 200 gOHM figure already factors in a 15% overfund, we need ~175 gOHM in the pool. 

50% overfund will be ~263 gOHM. Will do that. It'll create a juicy MEV opportunity initially, but worthwhile to prevent `forceSwap()` failing.

Will send the remainder to the TC timelock